legacy/JavaScript/JavaScriptSDK/ajax/ajax.ts

```diff
@@ -85,7 +85,7 @@ module Microsoft.ApplicationInsights {
         private instrumentOpen() {
             var originalOpen = XMLHttpRequest.prototype.open;
             var ajaxMonitorInstance = this;
-            XMLHttpRequest.prototype.open = function (method, url, async) {
+            XMLHttpRequest.prototype.open = function (method, url, async?) {
```

Comment:
Change backported from my TypeScript 3.7.x upgrade PR (Where I tested these changes) as the async is really optional.